nextflow.config

```diff
@@ -201,7 +201,7 @@ manifest {
   description = 'CAGE-seq pipeline'
   mainScript = 'main.nf'
   nextflowVersion = '>=21.04.0'
-  version = '2.0.0dev'
+  version = '2.0.0'
```

Comment:
are you sure this shouldn't still be `dev`?

Reply:
Not if we release it, right?
Do we want to include anything else, or should we try to release the 2.0.0 version?

Reply:
Ah, yes, sorry. I always forget the order of when to switch 🙃